WordPress/gutenberg
# Font Library: fix fonts not displaying correctly

This PR fixes #54706.

**Background**
There are two issues occurring, specific to font families with numbers and spaces inside their names / slugs:

1. The presets / CSS variable names and associated styles are output inconsistently between the editor and front-end
2. Font family names that include numbers and spaces are currently not wrapped in quotes, resulting in invalid CSS and the font family not being applied as expected.

**Solution**

Alter the `sanitize` method of the Font Family Class to:

1. Wrap slugs in `_wp_to_kebab_case`
2. Wrap font family names in quotes

Part of the solution includes a small bug fix in the `getIntersectingFontFaces`, as well as adding a test case.

**How to test**
See #54706 — test all reported fonts, in the site editor, post editor, and front end. Verify they appear as expected.
